app/src/main/java/live/hms/app2/ui/meeting/activespeaker/HlsFragment.kt

```diff
@@ -121,6 +124,23 @@ class HlsFragment : Fragment() {
         }
     }
 
+    override fun onPause() {
+        super.onPause()
+        playerEventsManager?.removeListener()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        if (isStatsActive) {
+            playerEventsManager?.removeListener()
```

Comment:
Better to move adding and removing of listener to MeetingViewModel so that we can interact using liveData 

For now its fine